.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,19 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches: [ master, CI_action ]
+  pull_request:
+    branches: [ master, CI_action ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: to_carts_repo
+      run: cd ../../
```

Comment:
@MCJOHN974 I believe this is redundant. Try to remove lines 16 and 17 and you will see that nothing changes.